test/validation/api/atomic/atomic.c

```diff
@@ -369,6 +379,44 @@ static void test_atomic_cas_dec_64(void)
 	}
 }
 
+static void test_atomic_cas_inc_128(void)
+{
+	int i, ret;
+	odp_u128_t old, new;
+	odp_atomic_u128_t *a128u = &global_mem->a128u;
+
+	old = a128u->v;
```

Comment:
Please add `odp_barrier_wait(&global_mem->global_barrier);` here.

Also, now when we have `odp_atomic_load_u128()` the test can follow the same model as `test_atomic_cas_inc_32()` and `test_atomic_cas_inc_64`.

Reply:
I will add the odp_barrier_wait(&global_mem->global_barrier); to this test as well as the  test_atomic_cas_dec_u128() and test_atomic_non_relaxed_u128(). Thanks!